### PR TITLE
doc-2596

### PR DESCRIPTION
**Pull request by aimurphy**
Describe that you can use any SA, not necessarily one associated with a delegate.
Some small fixes for whitespace, straight quotes, and heading levels

### CODE REVIEW

File: docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md [hunk 1]
```diff
@@ -101,107 +100,103 @@ roleRef:
   name: sa-role  
   apiGroup: rbac.authorization.k8s.io
 ```
-#### Builds (CI)
+### Builds (CI)
 
 A Kubernetes service account with CRUD permissions on Secret, Service, Pod, and PersistentVolumeClaim (PVC).
 
-For more information, see [User-Facing Roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles) from Kubernetes.
+For more information, see [User-Facing Roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles) from Kubernetes.
```
Comment (brian-f-harness) [inline docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md:107]:
```suggestion
For more information, go to [User-Facing Roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles) from Kubernetes.
```

File: docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md [hunk 2]
```diff
@@ -101,107 +100,103 @@ roleRef:
...
 See [Tags Reference](../../../20_References/tags-reference.md).
 
-### Cluster Details
+## Cluster Details
 
-#### Manual or Use a Delegate
+### Manual or Use a Delegate
 
-**Recommended:** Install and run the Harness Kubernetes Delegate in the target Kubernetes cluster, and then use the Kubernetes Cluster Connector to connect to that cluster using the Harness Kubernetes Delegate you installed. This is the easiest method to connect to a Kubernetes cluster.You can select to enter the authentication details of the target cluster or use the role associated with a Harness Delegate.
+**Recommended:** Install and run the Harness Kubernetes Delegate in the target Kubernetes cluster, and then use the Kubernetes Cluster Connector to connect to that cluster using the Harness Kubernetes Delegate you installed. This is the easiest method to connect to a Kubernetes cluster.You can select to enter the authentication details of the target cluster or use the role associated with a Harness Delegate.
```
Comment (brian-f-harness) [inline docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md:163]:
```suggestion
**Recommended:** Install and run the Harness Kubernetes Delegate in the target Kubernetes cluster, and then use the Kubernetes Cluster Connector to connect to that cluster using the Harness Kubernetes delegate you installed. This is the easiest method to connect to a Kubernetes cluster.You can select to enter the authentication details of the target cluster or use the role associated with a Harness Delegate.
```

File: docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md [hunk 3]
```diff
@@ -101,107 +100,103 @@ roleRef:
...
 
-When you select a Delegate, the Harness Delegate will inherit the Kubernetes service account associated with the Delegate pod.
+When you select a Delegate, the Harness Delegate inherits the Kubernetes service account associated with the Delegate pod.
```
Comment (brian-f-harness) [inline docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md:165]:
```suggestion
When you select a delegate, the Harness Delegate inherits the Kubernetes service account associated with the delegate pod.
```

File: docs/platform/7_Connectors/Cloud-providers/add-a-kubernetes-cluster-connector.md [hunk 4]
```diff
@@ -91,7 +91,7 @@ Select one of the following:
 
 * **Specify master URL and credentials**:
 	+ You provide the Kubernetes master node URL. The easiest method to obtain the master URL is using kubectl: `kubectl cluster-info`.
-	+ Next, enter the **Service Account Key** or other credentials.
+	+ Next, enter the **Service Account Key** or other credentials. You can use any service account; the service account doesn't have to be attached to a delegate.
 * **Use the credentials of a specific Harness Delegate**: Select this option to have the Connector inherit the credentials used by the Harness Delegate running in the cluster. You can install a Delegate as part of adding this Connector.
```
Comment (brian-f-harness) [inline docs/platform/7_Connectors/Cloud-providers/add-a-kubernetes-cluster-connector.md:95]:
```suggestion
* **Use the credentials of a specific Harness Delegate**: Select this option to have the Connector inherit the credentials used by the Harness Delegate running in the cluster. You can install a delegate as part of adding this Connector.
```

File: docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md [hunk 5]
```diff
@@ -16,15 +16,15 @@ For cloud platform-specific connections, see platform [Cloud Connectors](https:/
 
 Looking for the How-to? See [Add a Kubernetes Cluster Connector](https://developer.harness.io/docs/platform/Connectors/Cloud-providers/add-a-kubernetes-cluster-connector).
 
-### Video Summary
+## Video Summary
 
 Here's a 10min video that walks you through adding a Harness Kubernetes Cluster Connector and Harness Kubernetes Delegate. The Delegate is added to the target cluster and then the Kubernetes Cluster Connector uses the Delegate to connect to the cluster:
```
Comment (brian-f-harness) [inline docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md:21]:
```suggestion
Here's a 10min video that walks you through adding a Harness Kubernetes Cluster Connector and Harness Kubernetes delegate. The delegate is added to the target cluster and then the Kubernetes Cluster Connector uses the delegate to connect to the cluster:
```

File: docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md [hunk 6]
```diff
@@ -53,15 +53,14 @@ The Role used must have either the `cluster-admin` permission in the target clus
 
 For a detailed list of roles and policies, see [Harness Role-Based Access Control Overview](../../../4_Role-Based-Access-Control/1-rbac-in-harness.md#role).
 
-#### Harness CI Permission Requirements
+### Harness CI Permission Requirements
 
 If you are only using the Kubernetes Cluster Connector for Harness Continuous Integration (CI), you can use a reduced set of permissions.
 
 For Harness CI, the Delegate requires CRUD permissions on Secret and Pod.
```
Comment (brian-f-harness) [inline docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md:60]:
```suggestion
For Harness CI, the delegate requires CRUD permissions on Secret and Pod.
```

File: docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md [hunk 7]
```diff
@@ -101,107 +100,103 @@ roleRef:
...
-### Tags
+## Tags
 
 See [Tags Reference](../../../20_References/tags-reference.md).
 
-### Cluster Details
+## Cluster Details
 
-#### Manual or Use a Delegate
+### Manual or Use a Delegate
```
Comment (brian-f-harness) [inline docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md:161]:
```suggestion
### Manual or Use a delegate
```

File: docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md (hunk 2 above)
Comment (brian-f-harness) [inline docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md:163]:
```suggestion
**Recommended:** Install and run the Harness Kubernetes delegate in the target Kubernetes cluster, and then use the Kubernetes Cluster Connector to connect to that cluster using the Harness Kubernetes delegate you installed. This is the easiest method to connect to a Kubernetes cluster.You can select to enter the authentication details of the target cluster or use the role associated with a Harness Delegate.
```

File: docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md [hunk 8]
```diff
@@ -101,107 +100,103 @@ roleRef:
...
+**Recommended:** Install and run the Harness Kubernetes Delegate in the target Kubernetes cluster, and then use the Kubernetes Cluster Connector to connect to that cluster using the Harness Kubernetes Delegate you installed. This is the easiest method to connect to a Kubernetes cluster.You can select to enter the authentication details of the target cluster or use the role associated with a Harness Delegate.
 
-When you select a Delegate, the Harness Delegate will inherit the Kubernetes service account associated with the Delegate pod.
+When you select a Delegate, the Harness Delegate inherits the Kubernetes service account associated with the Delegate pod.
 
-The service account associated with the Delegate pod must have the Kubernetes `cluster-admin` role.
+The service account associated with the Delegate pod must have the Kubernetes `cluster-admin` role.
```
Comment (brian-f-harness) [inline docs/platform/7_Connectors/Cloud-providers/ref-cloud-providers/kubernetes-cluster-connector-settings-reference.md:167]:
```suggestion
The service account associated with the delegate pod must have the Kubernetes `cluster-admin` role.
```

**Review by brian-f-harness — APPROVED** [review]
LGTM. Added a couple suggestions, mostly for: https://harness.atlassian.net/wiki/spaces/DOCS/pages/21130772574/Documentation+style+guide#Platform. Some references were outside of changes you made.

File: docs/platform/7_Connectors/Cloud-providers/add-a-kubernetes-cluster-connector.md (hunk 4 above)
Comment (aimurphy) [reply docs/platform/7_Connectors/Cloud-providers/add-a-kubernetes-cluster-connector.md:95]: These two topics need full refreshes. I am just trying to close an old CI ticket right now.

**Pull request merged**